boards/espressif/esp_wrover_kit/doc/index.rst

```diff
@@ -43,7 +43,7 @@ Functional Description
 The following two figures and the table below describe the key components, interfaces, and controls
 of the ESP-WROVER-KIT board.
 
-.. figure:: img/esp-wrover-kit-v4.1-layout-front.jpg
+.. figure:: img/esp_wrover_kit.jpg
```

Comment:
@kartben Sounds you only renamed the image file.  Was the image re-worked as well?